sgf-viewer/src/content.ts

```diff
@@ -284,6 +284,68 @@ export const pages: object =
+            },
+            {
+                "title": "Human exploits Leela Zero",
+                "dir_name": "human_vs_lz100k",
+                "server": "dqn.ist.berkeley.edu",
+                "paths": [
+                    "/nas/ucb/kpelrine/humanatk/humanattack-LZ-100kvisits.sgfs"
+                ],
+                "max_games": 10,
+                "adversary": "Kellin Pelrine (Author)",
+                "victim": "Leela Zero, 100K visits",
+                "description": ["The same Go expert (Kellin Pelrine) also exploited Leela Zero with 100K visits, which would likewise normally be superhuman."]
+            },
+            {
+                "title": "Human exploits a top KGS bot",
+                "dir_name": "human_vs_jbxkata005",
+                "server": "dqn.ist.berkeley.edu",
+                "paths": [
+                    "/nas/ucb/kpelrine/humanatk/vs-JBXKata005/corners-JBXKata005-2.sgfs",
+                    "/nas/ucb/kpelrine/humanatk/vs-JBXKata005/corners-JBXKata005-3.sgfs",
+                    "/nas/ucb/kpelrine/humanatk/vs-JBXKata005/corners-JBXKata005-4.sgfs",
+                    "/nas/ucb/kpelrine/humanatk/vs-JBXKata005/corners-JBXKata005-5.sgfs",
+                    "/nas/ucb/kpelrine/humanatk/vs-JBXKata005/corners-JBXKata005-6.sgfs",
+                    "/nas/ucb/kpelrine/humanatk/vs-JBXKata005/corners-JBXKata005-7.sgfs",
+                    "/nas/ucb/kpelrine/humanatk/vs-JBXKata005/corners-JBXKata005-8.sgfs",
+                    "/nas/ucb/kpelrine/humanatk/vs-JBXKata005/corners-JBXKata005-9.sgfs",
+                    "/nas/ucb/kpelrine/humanatk/vs-JBXKata005/JBXKata005-corners.sgfs",
+                    "/nas/ucb/kpelrine/humanatk/vs-JBXKata005/JBXKata005-corners-2.sgfs",
+                    "/nas/ucb/kpelrine/humanatk/vs-JBXKata005/JBXKata005-corners-3.sgfs",
+                    "/nas/ucb/kpelrine/humanatk/vs-JBXKata005/JBXKata005-corners-4.sgfs",
+                    "/nas/ucb/kpelrine/humanatk/vs-JBXKata005/JBXKata005-corners-5.sgfs",
+                    "/nas/ucb/kpelrine/humanatk/vs-JBXKata005/JBXKata005-corners-6.sgfs",
+                    "/nas/ucb/kpelrine/humanatk/vs-JBXKata005/corners-JBXKata005.sgfs"
+                ],
+                "max_games": 15,
+                "adversary": "Kellin Pelrine (Author)",
+                "victim": "JBXKata005, 9 dan on KGS",
+                "description": ["Playing under standard human conditions on the online Go server KGS, the same Go expert (Kellin Pelrine) successfully exploited the bot JBXKata005 in 14/15 games. In the remaining game, the cyclic group attack still led to a successful capture, but the victim had enough points remaining to win. This bot uses a custom KataGo implementation, and at the time of the games was the strongest bot available to play and ranks #4 of all accounts on KGS."]
```

Comment:
Thanks for the explanation on why others bot weren't available to play / unclear status of accounts. I think given the complexity of this on balance less confusing if you just remove the "ranks #4 of all accounts".

Reply:
Makes sense. Done.